cloudbuild/python-scripts/utils/utils.py

```diff
@@ -0,0 +1,184 @@
+"""Python class containing basic utilities required for the creation of BQ Table.
+
+Can create specified number of rows, transfer them to BQ, delete the local file.
+"""
+
+import datetime
+import os
+import re
+import avro
+import avro.datafile
+import avro.io
+from google.cloud import bigquery
+
+
+class TableCreationUtils:
+    """Class containing all the basic utilities for creation of BQ table.
```

Comment:
`Utilities for creation of BQ table.` is sufficient for opening sentence.

Reply:
Done.